src/Features/Homescreen.php

```diff
@@ -74,18 +75,68 @@ public function add_user_data_fields( $user_data_fields ) {
 	 * Registers home page.
 	 */
 	public function register_page() {
+		// Register a top-level item for users who cannot view the core WooCommerce menu.
+		if ( ! $this->is_admin_user() ) {
+			add_filter(
+				'woocommerce_prevent_admin_access',
+				function( $bool ) {
+					// phpcs:disable WordPress.Security.NonceVerification.Recommended
+					if ( isset( $_GET['page'] ) && 'wc-admin' === $_GET['page'] ) {
```

Comment:
Do we also want to add `wc-reports` to this, as acceptable? And probably use `self::MENU_SLUG` here again for the `$_GET['page']`.